client/tests/unit/analyze-pending.spec.ts

```diff
@@ -1,16 +1,24 @@
 import { createLocalVue, mount } from '@vue/test-utils'
 import Vuetify from 'vuetify'
-import Comp from '@/components/new-request/analyze-pending.vue'
+import AnalyzePending from '@/components/new-request/analyze-pending.vue'
+import newReqModule from '@/store/new-request-module'
 
 const localVue = createLocalVue()
 const vuetify = new Vuetify()
 
 localVue.use(Vuetify)
 
 describe('analyze-pending.vue', () => {
-  const wrapper = mount(Comp, {
-    localVue,
-    vuetify
+  let wrapper: any
+  beforeAll(async () => {
+    newReqModule.mutateEntityType('CR')
+    newReqModule.mutateLocation('BC')
+    newReqModule.mutateRequestAction('NEW')
+    newReqModule.mutateAnalyzePending(true)
+    wrapper = mount(AnalyzePending, {
+      localVue,
+      vuetify
+    })
```

Comment:
If it works... then it works and no need to change anything...

But in this case, to be explicit, I would have "awaited" each of the mutate statements.

Also, usually, in unit tests, we need to `await Vue.nextTick()` after mounting a component. This is needed to give Vue time to asynchronously finish whatever the component needs to do when it's mounted, before we can verify any elements or variables.

Reply:
ah okay I will keep this in mind for next time